src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DisposeObjectsBeforeLosingScopeTests.cs

```diff
@@ -12218,5 +12218,40 @@ End Sub
 End Class",
                 GetBasicResultAt(12, 18, "New A()"));
         }
+
+        [Fact]
+        public async Task Dispose_UsingDeclaration_NoDiagnostic()
```

Comment:
If I got things right from the ticket on roslyn's side these are the extra cases what used to be causing issues.